### ISSUE TITLE
Oct 2018: Blog

### ISSUE THREAD

**Issue by howtohumanbetter** [seekhealing/seekhealing.github.io#164]
First three posts:
- A Spark Was Lit by Dr. Rachel Wurzman (please take content from https://seekhealing.org/a-spark-was-lit/

- How to Capitalize on the Power of Connection by Jennifer Nicolaisen (https://medium.com/@jen.the.human/how-to-capitalize-on-the-power-of-connection-3c56d65b9c00 )

- Why Capital Punishment for Drug Dealers will Only Make the Opioid Crisis Worse by Jennifer Nicolaisen ( https://medium.com/@jen.the.human/why-capital-punishment-for-drug-dealers-will-only-make-the-opioid-crisis-worse-ea81fb5eb8ea )

**Comment by shredtechular** [comment]
@howtohumanbetter  So, for this one, I'm assuming we need to build out this blogging functionality as we don't have it currently... Is that right or am I misunderstanding this one?

**Comment by howtohumanbetter** [comment]
@shredtechular that is right -- I recall you had some ideas about how to do this that we should probably revisit. Also want to discuss the functionality and how we will upload new posts in the future. 

Let's table this as a lower priority though, it's not necessary for Monday. Trying to figure out how to add labels to these so you can see priorities!
